op-node/rollup/derive/channel_out.go

```diff
@@ -150,6 +151,12 @@ func (co *ChannelOut) OutputFrame(w *bytes.Buffer, maxSize uint64) (uint16, erro
 		FrameNumber: uint16(co.frame),
 	}
 
+	// Error if the `maxSize` is too small to fit the fixed frame
+	// overhead as specified below.
+	if maxSize < 23 {
```

Comment:
23 should be a named constant (something like FrameV0OverHeadSize)

Reply:
Good catch, thanks!